views-fieldset-tck/src/main/java/io/micronaut/views/fields/tck/SelectViewRenderTest.java

```diff
@@ -22,10 +22,10 @@
 import io.micronaut.views.ViewsRenderer;
 import io.micronaut.views.fields.Fieldset;
 import io.micronaut.views.fields.FieldsetGenerator;
-import io.micronaut.views.fields.Message;
-import io.micronaut.views.fields.Option;
-import io.micronaut.views.fields.OptionFetcher;
-import io.micronaut.views.fields.SelectFormElement;
+import io.micronaut.views.fields.message.Message;
+import io.micronaut.views.fields.elements.Option;
+import io.micronaut.views.fields.fetcher.OptionFetcher;
+import io.micronaut.views.fields.elements.SelectFormElement;
```

Comment:
Nitpick! 🦟

Consistency:

`elements` is plural
`fetcher` and `message` are singular

Do we want all plural or all singular?

Reply:
I am not sure packages have to follow a naming convention but I [changed it to `fetchers` and `messages`](https://github.com/micronaut-projects/micronaut-views/pull/613/commits/b773e06c506618d3eb4e5c804ed572eb560ef543)

Reply:
Cool, I agree we don't need a convention, but the inconsistency was jarring to my brain 🧠